### PR TITLE
Correct descriptions of failed examples in valid language tag rule

### PR DESCRIPTION
**Pull request by sabernhardt**
The first two failed examples do not have a known primary language tag, and the third example could say "three-letter code."

Need for Call for Review:
This will not require a Call for Review: editorial changes (examples section)

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/html-page-lang-valid-bf051a.md [hunk 1]
```diff
@@ -90,23 +90,23 @@ This `html` element has a `lang` attribute with a [known primary language tag][]
 
 #### Failed Example 1
 
-This `html` element has a `lang` attribute with a [known primary language tag][].
+This `html` element has a `lang` attribute, but it is not a [known primary language tag][].
```
Comment (Jym77) [inline _rules/html-page-lang-valid-bf051a.md]:
```suggestion
This `html` element has a `lang` attribute, with no [known primary language tag][].
```
The value of the `lang` attribute is not (and does not need to be) a primary language tag, only the, well, primary tag needs to be.

File: _rules/html-page-lang-valid-bf051a.md [hunk 2]
```diff
@@ -90,23 +90,23 @@ This `html` element has a `lang` attribute with a [known primary language tag][]
...
 
 ```html
 <html lang="em-US"></html>
 ```
 
 #### Failed Example 2
 
-This `html` element has a `lang` attribute with a [known primary language tag][].
+This `html` element has a `lang` attribute, but it is not a [known primary language tag][].
```
Comment (Jym77) [inline _rules/html-page-lang-valid-bf051a.md]:
```suggestion
This `html` element has a `lang` attribute, with no [known primary language tag][].
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Well spotted 👍 A bit of polishing.

**Review by giacomo-petri — APPROVED** [review]
Approving with @Jym77's changes.

**Review by carlosapaduarte — APPROVED** [review]
Committing the changes proposed by @Jym77 to try to move this one forward :-)

**Review by Jym77 — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
Merging.
Editorial changes, no call for review required.

**Pull request merged**